spec/api/courses_spec.rb

```diff
@@ -1,23 +1,17 @@
 require "swagger_helper"
 
 describe "API" do
-  path "/recruitment_cycles/{year}/providers/{provider_code}/courses" do
-    get "Returns the courses for the specified provider." do
-      operationId :public_api_v1_provider_courses
-      tags "course"
+  path "/recruitment_cycles/{year}/courses" do
+    get "Returns the courses for the specified year." do
```

Comment:
```suggestion
    get "Returns the courses for the specified recruitment cycle." do
```

based on 
https://api2.qa.publish-teacher-training-courses.service.gov.uk/api-reference.html#recruitment_cycles-year-providers-get